tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Test_BjorhusImpl.cpp

```diff
@@ -0,0 +1,848 @@
+
+  // 3-index constraint
+  tnsr::iaa<DataVector, VolumeDim, frame> local_three_index_constraint(
+      slice_grid_points, std::numeric_limits<double>::signaling_NaN());
+
+  {
+    // Setting coords
+    for (size_t i = 0; i < VolumeDim; ++i) {
+      local_inertial_coords.get(i) = inertial_coords.get(i);
+    }
+
+    // Setting local_unit_interface_normal_one_form
+    for (size_t i = 0; i < get<0>(local_inertial_coords).size(); ++i) {
+      local_unit_interface_normal_one_form.get(0)[i] = -1.;
+      local_unit_interface_normal_one_form.get(1)[i] = 1.;
+      local_unit_interface_normal_one_form.get(2)[i] = 1.;
+    }
+    // Setting local_spacetime_unit_normal_vector
+    for (size_t i = 0; i < get<0>(local_inertial_coords).size(); ++i) {
+      local_spacetime_unit_normal_vector.get(0)[i] = -1.;
+      local_spacetime_unit_normal_vector.get(1)[i] = -3.;
+      local_spacetime_unit_normal_vector.get(2)[i] = -5.;
+      local_spacetime_unit_normal_vector.get(3)[i] = -7.;
+    }
+    // Setting local_inverse_spatial_metric
+    for (size_t i = 0; i < get<0>(local_inertial_coords).size(); ++i) {
+      for (size_t j = 0; j < VolumeDim; ++j) {
+        local_inverse_spatial_metric.get(0, j)[i] = 41.;
+        local_inverse_spatial_metric.get(1, j)[i] = 43.;
+        local_inverse_spatial_metric.get(2, j)[i] = 47.;
+      }
+    }
+    // Setting local_spacetime_metric
+    for (size_t i = 0; i < get<0>(local_inertial_coords).size(); ++i) {
+      for (size_t a = 0; a <= VolumeDim; ++a) {
+        local_spacetime_metric.get(0, a)[i] = 257.;
+        local_spacetime_metric.get(1, a)[i] = 263.;
+        local_spacetime_metric.get(2, a)[i] = 269.;
+        local_spacetime_metric.get(3, a)[i] = 271.;
+      }
+    }
+    // Setting local_inverse_spacetime_metric
+    for (size_t i = 0; i < get<0>(local_inertial_coords).size(); ++i) {
+      for (size_t a = 0; a <= VolumeDim; ++a) {
+        local_inverse_spacetime_metric.get(0, a)[i] =
+            -277.;  // needs to be < 0 for lapse
+        local_inverse_spacetime_metric.get(1, a)[i] = 281.;
+        local_inverse_spacetime_metric.get(2, a)[i] = 283.;
+        local_inverse_spacetime_metric.get(3, a)[i] = 293.;
+      }
+    }
+    // Setting local_extrinsic_curvature
+    // ONLY ON THE +Y AXIS (Y = +0.5)
+    for (size_t i = 0; i < get<0>(local_inertial_coords).size(); ++i) {
+      std::array<double, 9> spec_vals{};
+      bool not_initialized = true;
+
+      if ((lic0[i] == 299. or lic0[i] == 299.5 or lic0[i] == 300.) and
+          lic1[i] == 0.5 and
+          (lic2[i] == -0.5 or lic2[i] == 0. or lic2[i] == 0.5)) {
+        spec_vals = {{200.2198037251189, 266.7930716334918, 333.3663395418648,
+                      266.7930716334918, 333.3663395418648, 399.9396074502377,
+                      333.3663395418648, 399.9396074502377, 466.5128753586106}};
+        not_initialized = false;
+      }
+
+      if (not_initialized) {
+        ASSERT(true,
```

Comment:
I think this is an `ERROR`---it's a big problem in release mode too :) Since it's in a test I'd say we're better off adding an `ERROR` and paying for the overhead of having the extra code than to have some silent test breakage

Reply:
fixed to throw an `ERROR`